src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +87,15 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * void fun1() {}    // violation, 4 whitespaces before the comment starts
+ * void fun2() { return; } /* violation here, 2 whitespaces before the comment starts *&#47;
```

Comment:
there is only one whitespace

Reply:
that was mistake while copy-pasting, fixing it.